channels.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/mackerelio/mackerel-client-go"
+	"github.com/mackerelio/mkr/format"
+	"github.com/mackerelio/mkr/logger"
+	"github.com/mackerelio/mkr/mackerelclient"
+	"github.com/urfave/cli"
+)
+
+var commandChannels = cli.Command{
+	Name:  "channels",
+	Usage: "List notification channels",
+	Description: `
+	Lists notification channels. With no subcommand specified, this will show all channels.
+	Requests APIs under "/api/v0/channels". See https://mackerel.io/api-docs/entry/channels .
+	`,
+	Action: doChannelsList,
+	Subcommands: []cli.Command{
+		{
+			Name:      "pull",
+			Usage:     "pull channel settings",
+			ArgsUsage: "[--file-path | -F <file>] [--verbose | -v]",
+			Description: `
+    Pull channels settings from Mackerel server and save them to a file. The file can be specified by filepath argument <file>. The default is 'channels.json'.
+`,
+			Action: doChannelsPull,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "file-path, F", Value: "", Usage: "Filename to store channel settings. default: channels.json"},
+				cli.BoolFlag{Name: "verbose, v", Usage: "Verbose output mode"},
+			},
+		},
+	},
+}
+
+func doChannelsList(c *cli.Context) error {
+	client := mackerelclient.NewFromContext(c)
+	channels, err := client.FindChannels()
+	logger.DieIf(err)
+
+	format.PrettyPrintJSON(os.Stdout, channels)
+	return nil
+}
+
+func doChannelsPull(c *cli.Context) error {
+	isVerbose := c.Bool("verbose")
+	filePath := c.String("file-path")
+
+	channels, err := mackerelclient.NewFromContext(c).FindChannels()
+	logger.DieIf(err)
+
+	channelSaveRules(channels, filePath)
+
+	if isVerbose {
+		format.PrettyPrintJSON(os.Stdout, channels)
+	}
+
+	if filePath == "" {
+		filePath = "channels.json"
```

Comment:
You are assigning the default value of `filePath` at here and in `channelSaveRules`. It may cause bugs in the future. I think if you move this assignment to the top of the L58 then the second assignment is become useless.